src/server/scripts/Spells/spell_shaman.cpp

```diff
@@ -866,6 +869,47 @@ class spell_sha_item_t18_elemental_4p_bonus : public SpellScriptLoader
     }
 };
 
+// 51505 - Lava burst
+class spell_sha_lava_burst : public SpellScriptLoader
+{
+    public:
+        spell_sha_lava_burst() : SpellScriptLoader("spell_sha_lava_burst") { }
+
+        class spell_sha_lava_burst_SpellScript : public SpellScript
+        {
+            PrepareSpellScript(spell_sha_lava_burst_SpellScript);
+
+            bool Validate(SpellInfo const* /*spellInfo*/) override
+            {
+                if (!sSpellMgr->GetSpellInfo(SPELL_SHAMAN_LAVA_BURST))
```

Comment:
Spell scripting basics: you do NOT need to check the existence of the spell being scripted, this is done by ObjectMgr when loading (and if it doesn't exist then Validate will not be called)